dashboard/config/foorm/forms/surveys/pd/summer_workshop_post_survey.0.json

```diff
@@ -171,118 +171,601 @@
+    },
+
+    {
+     "type": "panel",
+     "name": "In_person_panel",
+     "elements": [
+    
     {
      "type": "html",
      "name": "ifCs",
-     "html": "<p><span style=\"background-color: #ffff00;\">&gt;&gt;LOGISTICS AND TOOLS / VIRTUAL-SPECIFIC&lt;&lt;</span></p>\nCurrently a simple FR item (see below).  Could add virtual-specific items here"
+     "html": "<p><span style=\"background-color: #ffff00;\">&gt;&gt;IN-PERSON LOGISTICS AND TOOLS &lt;&lt;</span>"
     },
     {
+          "type": "matrix",
+          "name": "logistics_in_person",
+          "title": "To what extent are the following true about this workshop?",
+          "rows": [
+            {
+              "value": "I_received_clear_communic",
+              "text": "I received clear communication about when and where the workshop would take place."
+            },
+            {
+              "value": "The_workshop_venue_met_my",
+              "text": "The workshop venue met my learning needs."
+            }
+          ],
+          "columns": [
+            {
+              "value": "1",
+              "text": "Not at all"
+            },
+            {
+              "value": "2",
+              "text": "Somewhat"
+            },
+            {
+              "value": "3",
+              "text": "Mostly"
+            },
+            {
+              "value": "4",
+              "text": "To a great extent"
+            }
+          ],
+          "visible": true
+        },
+        {
      "type": "comment",
      "name": "venue_and_logistics_FR",
      "title": "Do you have feedback about the venue and the way logistics were run for the workshop this week? Please be specific and provide suggestions for improvement.",
      "cols": 60,
      "rows": 3
+    }
+    ],
+     "visibleIff": "{is_virtual} = 'false'"
```

Comment:
should be visibleIf (there's 4 places this needs to be updated--also on line 316, 659, 767)

Reply:
🤦  fixed.